File: CoreJava/game-demo-app/src/com/techlab/game/Game.java
package com.techlab.game;

import java.util.Random;
import java.util.Scanner;

public class Game {

	private int systemNumber;

	public boolean startGame(int systemNumber, int userNumber ) {
		if (systemNumber > userNumber) {
			System.err.println("Your guess is low");
			return true;
		} else if (systemNumber < userNumber) {
			System.err.println("Your guess is high");
			return true;
		} else {
			System.out.println("Your guess is right....!");
			return false;
		}
	}

}

File: C#Basics/GuessTheNumber/Game.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GuessTheNumber
{
    class Game
    {
		private int systemNumber;

		public bool startGame(int systemNumber, int userNumber)
		{
			if (systemNumber > userNumber)
			{
				Console.WriteLine("Your guess is low");
				return true;
			}
			else if (systemNumber < userNumber)
			{
				Console.WriteLine("Your guess is high");
				return true;
			}
			else
			{
				Console.WriteLine("Your guess is right....!");
				return false;
			}
		}
	}
}
